charts/gitprime-app/v3.0.1/values.yaml

```diff
@@ -14,6 +14,9 @@ dataPipeline:
   datadog:
     enabled: false
 
+  dns:
+    ndots: 5
```

Comment:
Shouldn't the default value be 2?

Reply:
I don't have any problems setting it to 2.  Let me update that now